features/flows/elastic/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -26,13 +26,6 @@
             <cm:property name="connTimeout" value="5000" /> <!-- 5 second timeout for Elasticsearch operations -->
             <cm:property name="readTimeout" value="30000" /> <!-- 30 second timeout for Elasticsearch socket reads -->
             <cm:property name="retries" value="0" /> <!-- Disable retries by default -->
-            <cm:property name="enableForwarding" value="false" /> <!-- Disable flow forwarding -->
-            <cm:property name="nodeCache.enabled" value="true" /> <!-- Set to false to disable caching -->
-            <cm:property name="nodeCache.maximumSize" value="10000"/> <!-- Set value for unlimited size -->
-            <cm:property name="nodeCache.expireAfterWrite" value="0"/> <!-- in seconds. Set to 0 to never evict elements -->
-            <cm:property name="nodeCache.recordStats" value="true"/> <!-- Set to false to not expose cache statistics via jmx -->
-            <cm:property name="flowThreshold.stepSizeMs" value="300000"/>
-            <cm:property name="flowThreshold.idleTimeoutMs" value="900000"/>
```

Comment:
Can we document (even if it's just in the release notes) why these are removed, and what the alternatives are now, or why they are no longer necessary?

Reply:
They moved to another `blueprint.xml` (https://github.com/OpenNMS/opennms/pull/5040/files#diff-229201c368c8b1e5256d8d761b77208627803535b7c6b459309a12704451ad98) but are still available under the same name.